x-pack/plugins/uptime/server/graphql/monitor_states/resolvers.ts

```diff
@@ -32,18 +33,24 @@ export const createMonitorStatesResolvers: CreateUMGraphQLResolvers = (
       async getMonitorStates(
         _resolver,
         { dateRangeStart, dateRangeEnd, filters, pagination, statusFilter },
-        { APICaller }
+        { APICaller, savedObjectsClient }
       ): Promise<MonitorSummaryResult> {
+        const dynamicSettings = await savedObjectsAdapter.getUptimeDynamicSettings(
+          savedObjectsClient,
+          undefined
```

Comment:
why you have to pass undefined?

Reply:
i see the same case in other places as well.

Reply:
Good point! The type required a second argument for no reason.